rubocop.yml.tt

```diff
@@ -100,3 +101,9 @@ Style/StringLiterals:
 
 Style/TrivialAccessors:
   AllowPredicates: true
+
+RSpec/MultipleExpectations:
+  Max: 10
+
+RSpec/ExampleLength:
+  Max: 20
```

Comment:
I'm going to pull these out into the dotfiles repo

Reply:
Turns out that breaks older installations of rubocop because they choke on the unexpected `Rspec/` group so I'm putting it back in here